cmd/chloggen/entry.go

```diff
@@ -67,5 +69,15 @@ func (e Entry) String() string {
 		issueStrs = append(issueStrs, fmt.Sprintf("#%d", issue))
 	}
 	issueStr := strings.Join(issueStrs, ", ")
-	return fmt.Sprintf("- `%s`: %s (%s)", e.Component, e.Note, issueStr)
+
+	var sb strings.Builder
+	sb.WriteString(fmt.Sprintf("- `%s`: %s (%s)", e.Component, e.Note, issueStr))
+	if e.SubText != "" {
+		if runtime.GOOS == "windows" {
+			sb.WriteString("\r")
+		}
```

Comment:
This will produce a different file when running on Windows, I don't think we want that, right?

Reply:
Ultimately I'm just trying to pass the tests on windows, but I don't know these tests really need to run there. I'm leaning towards just skipping these tests on windows.